clc/cmd/clc.go

```diff
@@ -53,12 +54,15 @@ func NewMain(arg0, cfgPath, logPath, logLevel string, stdout, stderr io.Writer)
 		CompletionOptions: cobra.CompletionOptions{DisableDescriptions: true},
 		SilenceErrors:     true,
 	}
+	rc.SetOut(sio.Stdout)
+	rc.SetErr(sio.Stderr)
```

Comment:
Should we set `rc.SetIn(sio.Stdin)` too?

Reply:
No, we don't need it.